CMakeLists.txt

```diff
@@ -5,16 +5,20 @@ enable_testing()
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 message(STATUS "CMAKE_MODULE_PATH: ${CMAKE_MODULE_PATH}")
 
-find_package(Aio REQUIRED)
```

Comment:
I would like remain `find_package` here. We just don't compile things require libaio if we couldn't find one. This is more friendly to users. We could emit a macro in CMakeLists to detect if we find libaio.

Reply:
OK